### ISSUE TITLE
[BUG] fvm flutter run not working on any device.

### ISSUE THREAD

**Issue by asfandalavi75** [fluttertools/fvm#436]
**Describe the bug**
When I run fvm flutter run, if I choose chrome as my device, I just get an endless **'Waiting for connection from debug service on Chrome...'** message, and a blank localhost page. When I use android I get **"Could not get unknown property 'compileSdkVersion' for extension 'flutter' of type FlutterExtension."**  in this location "project\android\app\build.gradle' line: 29".
I am using flutter **version 2.5.3** with fvm.  

**To Reproduce**

1. Go to terminal..
2. Run `fvm flutter run`...
3. Check...

- OS: [Windows 10]
- FVM Version [2.3.1]

**Additional context**
Before this I was having an issue with the pubspec.yaml file, where I had to change the dart sdk version from sdk: ">=2.17.3 <3.0.0" to sdk: ">=2.14.4 <3.0.0" and also flutter_lints: ^2.0.0 to flutter_lints: ^1.0.4 because of a dependency issue.

**Comment by leoafarias** [comment]
@asfandalavi75 can you confirm that the issue only happens with FVM?

**Comment by asfandalavi75** [comment]
> @asfandalavi75 can you confirm that the issue only happens with FVM?

Yes, before yesterday I was on the latest version of flutter and everything was working fine.

**Comment by leoafarias** [comment]
@asfandalavi75 have you tried reinstalling fvm after upgrading dart?

**Comment by asfandalavi75** [comment]
A day before i installed fvm, I had already run flutter upgrade, which i believe updates flutter and the dart sdk. As i mentioned in "additional context", I believed that the initial problem was that flutter version 2.5.3(which is the version i was wanted to use with the help of fvm) wasn't compatible with the lastest dark sdk, so i set the version in pubspec to yaml to a supported version.

**Comment by asfandalavi75** [comment]
Just checked, I already had the latest version of dart installed before i tried to install fvm.

**Comment by leoafarias** [comment]
ok so I believe this is not an issue on FVM, I think you might have to configure the Dart path to get it from the Flutter dart path itself and not from the separate Dart standalone install

**Comment by asfandalavi75** [comment]
My dart sdk is already inside my flutter sdk file.

**Comment by asfandalavi75** [comment]
I have also tried setting my global flutter version to the version i downloaded using fvm, and I have updated its path as well, but I'm having the same problem:  Could not get unknown property 'compileSdkVersion' for extension 'flutter' of type FlutterExtension

**Comment by asfandalavi75** [comment]
Ok so I made a new project using the flutter version i downloaded using fvm(I set that version as my global version), and now everything seems to be working fine. The debug version of my code is compiling and I am getting no errors.
